src/zosmf_workflows/zowe/zosmf_workflows_for_zowe_sdk/workflows.py

```diff
@@ -0,0 +1,557 @@
+This program and the accompanying materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this distribution, and is available at
+
+https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zowe Project.
+
+Contributors:
+    Zowe Community
+    Uladzislau Kalesnikau
+"""
+
+from typing import Any, Literal, Optional
+
+from zowe.core_for_zowe_sdk import SdkApi
+
+from .response import (
+    CreateWorkflowResponse,
+    GetArchivedWorkflowPropertiesResponse,
+    GetWorkflowDefinitionResponse,
+    GetWorkflowPropertiesResponse,
+    ListArchivedWorkflowsResponse,
+    ListWorkflowsResponse,
+)
+
+
+class Workflows(SdkApi):
+    """
+    Representation of the base z/OSMF Workflows API.
+    
+    See more at https://www.ibm.com/docs/en/zos/3.1.0?topic=services-zosmf-workflow.
+
+    Parameters
+    ----------
+    connection: dict[str, Any]
+        The connection object
+    version: str
+        The supported version of z/OSMF Workflows (1.0 is the only version available for now)
+    """
+    
+    def __init__(self, connection: dict[str, Any], version: str = "1.0"):
+        super().__init__(connection, "/zosmf/workflow/rest/{}".format(version))
+
+    def _step_variables_body(
+        self, return_steps_data: bool, return_variables_data: bool
+    ) -> dict[str, Any]:
+        """
+        Form steps-variables request body.
+
+        If `return_steps_data` is true, "steps" body parameter is added.
+        If `return_variables_data` is true, "variables" body parameter is added.
+        If both parameters are true, "steps,variables" body parameter is added.
+        Otherwise it returns an empty dictionary
+        """
+        params = {}
+        if return_steps_data:
+            params["returnData"] = "steps"
+            if return_variables_data:
+                params["returnData"] += ",variables"
+        elif return_variables_data:
+            params["returnData"] = "variables"
+        return params
+
+    def create_workflow(
+        self,
+        workflow_name: str,
+        workflow_definition_file: str,
+        system: str,
+        owner: str,
+        workflow_definition_file_system: Optional[str] = None,
+        variable_input_file: Optional[str] = None,
+        variables: Optional[list[dict]] = None,
+        resolve_global_conflict_by_using: Literal['global', 'input'] = "global",
+        workflow_archive_safid: Optional[str] = None,
+        comments: Optional[str] = None,
+        assign_to_owner: bool = True,
+        access_type: Literal['Public', 'Restricted', 'Private'] = "Public",
+        account_info: Optional[str] = None,
+        job_statement: Optional[str] = None,
+        delete_completed_jobs: bool = False,
+        jobs_output_directory: Optional[str] = None,
+        auto_delete_on_completion: bool = False,
+        target_systemuid: Optional[str] = None,
+        target_systempwd: Optional[str] = None
```

Comment:
Thanks for keeping the similar defaults as the Nodejs SDK and the z/osmf REST API. 🙏 
I do wonder about these two properties in particular.

I know these are part of the request body, but curious to get your thoughts on the other endpoints that support actions on remote systems. (see [X-IBM-Target-System](https://www.ibm.com/docs/en/zos/3.1.0?topic=services-zos-data-set-file-rest-interface#izuhpinfo_api_restfiles__title__5)) 😋